settings.gradle

```diff
@@ -175,7 +175,7 @@ include ':instrumentation:servlet:glassfish-testing'
 include ':instrumentation:servlet:servlet-common'
 include ':instrumentation:servlet:servlet-2.2'
 include ':instrumentation:servlet:servlet-3.0'
-include ':instrumentation:spark-web-framework-2.3'
+include ':instrumentation:spark-2.3'
```

Comment:
`spark` makes me think Apache Spark. `spark-web-framework` name has it's problems too though. was thinking maybe `spark-web`, but that name has it's problems too. ok, I like `spark`. it seems popular enough for us to justify using that name, and Apache Spark would be `apache-spark` (like `apache-camel`) anyways if we even ever.